src/main/java/uk/gov/hmcts/cmc/claimstore/services/ccd/CoreCaseDataService.java

```diff
@@ -62,7 +66,7 @@ public CaseDetails save(String authorisation, Claim claim) {
                 .save(
                     authorisation,
                     eventRequestData,
-                    jsonMapper.toJson(ccdCase),
+                    toJson(ccdCase),
```

Comment:
Why can't we use `JsonMapper` @sabahirfan ?

Reply:
`jsonMapper` returns String object and if we pass `String` to `CCD Client` we are getting `400` Bad Request.

Reply:
Have you investigated why there's a 400 response? It should work with the `String`.

Reply:
Looks like it was due to double serialisation causing this by generating escaped characters.